_component_design/assisted-search.md

```diff
@@ -0,0 +1,120 @@
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assist search specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/search-assisted-default.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading state
+
+If autocomplete is taking time to load, a loading spinner should be used in place of results.
+
+User should still have instructions for how to search without autocompleted results (Enter).
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assist search loading state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/search-assisted-loading.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Search results
+
+Search completion results should be divided up by categories.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search - search results specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-assisted/search-assisted-results.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Assisted search states
+
+**Hover:** The user may click or use their arrow keys to select one of the items in the dropdown list.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search results specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-bar/search-assisted-states-1.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+**Active:** When a keyword is actively being used but has not yet been completed with search criteria.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Assisted search results - active state"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/search-and-filters/search-bar/search-assisted-states-2.svg)
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- **Filter + No Autocomplete:** Keyed entries that have not formed a complete query string should not have a background color.
```

Comment:
Do not use a background color if keyed entries have not formed a complete query string.

Reply:
updated.